### ISSUE TITLE
Are you still working on it?

### ISSUE THREAD

**Issue by Westixy** [TurboScriptLang/spec#3]
Hi guys!
The project looks awesome but dead too... are you still working on it or it is just on "pause" ?

**Comment by nidin** [comment]
Yes, of course. I am waiting for [WebGPU](https://en.wikipedia.org/wiki/WebGPU) spec to be standardised. Mean while you can check [AssemblyScript](https://github.com/AssemblyScript/assemblyscript). I am contributing to AS.
TurboScript will not be a TypeScript to WebAssembly compiler. Instead it will be a TypeScript dialect to SPIR-V compiler. At least that's my plan.
